llvm/include/llvm/SYCLLowerIR/ModuleSplitter.h

```diff
@@ -36,6 +37,23 @@ enum IRSplitMode {
   SPLIT_NONE        // no splitting
 };
 
+struct SplittedImage {
```

Comment:
I think `Module` over `Image` might be more consistent terminology here. Also, grammatically I think `Split` instead of `Splitted` here is more correct..

Reply:
done.